{
    "dataType": "CVE_RECORD",
    "dataVersion": "5.1",
    "cveMetadata": {
        "cveId": "CVE-2025-48943",
        "assignerOrgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
        "state": "PUBLISHED",
        "assignerShortName": "GitHub_M",
        "dateReserved": "2025-05-28T18:49:07.582Z",
        "datePublished": "2025-05-30T18:36:01.519Z",
        "dateUpdated": "2025-05-30T18:56:18.715Z"
    },
    "containers": {
        "cna": {
            "title": "vLLM allows clients to crash the openai server with invalid regex",
            "problemTypes": [
                {
                    "descriptions": [
                        {
                            "cweId": "CWE-248",
                            "lang": "en",
                            "description": "CWE-248: Uncaught Exception",
                            "type": "CWE"
                        }
                    ]
                }
            ],
            "metrics": [
                {
                    "cvssV3_1": {
                        "attackComplexity": "LOW",
                        "attackVector": "NETWORK",
                        "availabilityImpact": "HIGH",
                        "baseScore": 6.5,
                        "baseSeverity": "MEDIUM",
                        "confidentialityImpact": "NONE",
                        "integrityImpact": "NONE",
                        "privilegesRequired": "LOW",
                        "scope": "UNCHANGED",
                        "userInteraction": "NONE",
                        "vectorString": "CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H",
                        "version": "3.1"
                    }
                }
            ],
            "references": [
                {
                    "name": "https://github.com/vllm-project/vllm/security/advisories/GHSA-9hcf-v7m4-6m2j",
                    "tags": [
                        "x_refsource_CONFIRM"
                    ],
                    "url": "https://github.com/vllm-project/vllm/security/advisories/GHSA-9hcf-v7m4-6m2j"
                },
                {
                    "name": "https://github.com/vllm-project/vllm/issues/17313",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://github.com/vllm-project/vllm/issues/17313"
                },
                {
                    "name": "https://github.com/vllm-project/vllm/pull/17623",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://github.com/vllm-project/vllm/pull/17623"
                },
                {
                    "name": "https://github.com/vllm-project/vllm/commit/08bf7840780980c7568c573c70a6a8db94fd45ff",
                    "tags": [
                        "x_refsource_MISC"
                    ],
                    "url": "https://github.com/vllm-project/vllm/commit/08bf7840780980c7568c573c70a6a8db94fd45ff"
                }
            ],
            "affected": [
                {
                    "vendor": "vllm-project",
                    "product": "vllm",
                    "versions": [
                        {
                            "version": ">= 0.8.0, < 0.9.0",
                            "status": "affected"
                        }
                    ]
                }
            ],
            "providerMetadata": {
                "orgId": "a0819718-46f1-4df5-94e2-005712e83aaa",
                "shortName": "GitHub_M",
                "dateUpdated": "2025-05-30T18:37:17.548Z"
            },
            "descriptions": [
                {
                    "lang": "en",
                    "value": "vLLM is an inference and serving engine for large language models (LLMs). Version 0.8.0 up to but excluding 0.9.0 have a Denial of Service (ReDoS) that causes the vLLM server to crash if an invalid regex was provided while using structured output. This vulnerability is similar to GHSA-6qc9-v4r8-22xg/CVE-2025-48942, but for regex instead of a JSON schema. Version 0.9.0 fixes the issue."
                }
            ],
            "source": {
                "advisory": "GHSA-9hcf-v7m4-6m2j",
                "discovery": "UNKNOWN"
            }
        },
        "adp": [
            {
                "metrics": [
                    {
                        "other": {
                            "type": "ssvc",
                            "content": {
                                "timestamp": "2025-05-30T18:54:23.948239Z",
                                "id": "CVE-2025-48943",
                                "options": [
                                    {
                                        "Exploitation": "none"
                                    },
                                    {
                                        "Automatable": "no"
                                    },
                                    {
                                        "Technical Impact": "partial"
                                    }
                                ],
                                "role": "CISA Coordinator",
                                "version": "2.0.3"
                            }
                        }
                    }
                ],
                "title": "CISA ADP Vulnrichment",
                "providerMetadata": {
                    "orgId": "134c704f-9b21-4f2e-91b3-4a467353bcc0",
                    "shortName": "CISA-ADP",
                    "dateUpdated": "2025-05-30T18:56:18.715Z"
                }
            }
        ]
    }
}